test_windows.js

```diff
@@ -0,0 +1,31 @@
+var Mocha = require('mocha')
+var fs = require('fs');
+var path = require('path');
+var semver = require('semver');
+
+
+var mocha = new Mocha();
+var testDir = 'build/test';
+fs.readdirSync(testDir).filter(function(file) {
+    return file.substr(-3) === '.js';
+}).forEach(function(file) {
+    mocha.addFile(
+        path.join(testDir, file)
+    );
+});
+
+process.env.GCLOUD_PROJECT = 0
+process.env.CLOUD_DEBUG_ASSERTIONS = 1
+
+const nodeVersion = /v(\d+\.\d+\.\d+)/.exec(process.version);
+if (!nodeVersion || nodeVersion.length < 2) {
+  console.error('unable to get node version');
+} else if (semver.satisfies(nodeVersion[1], '>=8')) {
```

Comment:
I understand this is the same logic as in `debuglet.ts` but I think it's unnecessary to parse the version string beforehand. `semver.satisfies(process.version, '>=8')` should just work here, and we don't need the first condition (that might even mask the issue because it doesn't exit with code 1).

Reply:
Sounds good. I will this elsewhere.